app/javascript/styles/fonts/montserrat.scss

```diff
@@ -0,0 +1,11 @@
+@font-face {
```

Comment:
It's not normal but if I leave anything in assets it will get picked up by sprockets, I think, and then it would get processed twice.